pkg/operator/clustermemberremovalcontroller/clustermemberremovalcontroller.go

```diff
@@ -77,16 +82,83 @@ func NewClusterMemberRemovalController(
 			masterNodeInformer,
 			masterMachineInformer,
 			kubeInformersForNamespaces.InformersFor(operatorclient.TargetNamespace).Core().V1().ConfigMaps().Informer(),
+			kubeInformersForNamespaces.InformersFor("kube-system").Core().V1().ConfigMaps().Informer(),
 		).ToController("ClusterMemberRemovalController", eventRecorder.WithComponentSuffix("cluster-member-removal-controller"))
 }
 
 func (c *clusterMemberRemovalController) sync(ctx context.Context, _ factory.SyncContext) error {
-	// stop if the machine API is not functional
+	// only attempt to scale down if the machine API is functional
 	if isFunctional, err := c.machineAPIChecker.IsFunctional(); err != nil {
 		return err
 	} else if !isFunctional {
 		return nil
 	}
+
+	var errs []error
+	if err := c.removeMemberWithoutMachine(ctx); err != nil {
```

Comment:
this refactor could be made early to reduce the diff while we argue over the rest.